docs/content/drafts/UnderlineNav2.mdx

```diff
@@ -111,6 +111,18 @@ If there is still overflow, the component will behave depending on the pointer.
 </UnderlineNav>
 ```
 
+### Loading State for Counters
+
+```jsx live drafts
+<UnderlineNav counterLoading={true}>
```

Comment:
How do you feel about renaming this prop to `loadingCounters`? I think it's a little more clear that applies to all the counters in the underline nav.

```suggestion
<UnderlineNav loadingCounters={true}>
```

Reply:
I debated it yesterday myself 😃 I wasn't sure but now you recommended it as well, so I'll update it, it makes more sense!